src/containers/BooksForm.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { categories } from '../util/bookCategories';
+import {createBook} from '../actions/index';
+import { connect } from 'react-redux';
+import {PropTypes} from 'prop-types';
+
+class BooksForm extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            title: '',
+            category: ''
```

Comment:
let's set the value of category in state to the first item as well. we are binding the input to it so let's have that first value as the default. also because after submitting the form, its is set to the first value again.

Reply:
Done